### PR TITLE
Adding/fixing docstrings and some tests

### PR DESCRIPTION
**Pull request by drvinceknight**
Docstrings in:

- match_generator
- tournament
- results_set

Tests: mainly just adding some extra checks for building (or not)
results set.

### CODE REVIEW

File: axelrod/match_generator.py [hunk 1]
```diff
@@ -64,10 +64,10 @@ def build_match_chunks(self, noise=0):
...
         round robin tournament.
 
-        Parameters
+        parameters
         ----------
         noise : float, 0
-            The probability that a player's intended action should be flipped
+            the probability that a player's intended action should be flipped
 
         Yields
```
Comment (marcharper) [inline axelrod/match_generator.py]: ¿por qué no ésta?

File: axelrod/tournament.py [hunk 2]
```diff
@@ -70,6 +70,14 @@ def play(self, build_results=True, filename=None):
         """
         Plays the tournament and passes the results to the ResultSet class
 
+        Parameters
+        ----------
+
```
Comment (marcharper) [inline axelrod/tournament.py]: spacing, and Parameters is capitalized (not sure what the guide is on this but the one above is changed)

**Comment by marcharper** [comment]
Feel free to merge on a successful travis build.

File: axelrod/match_generator.py (hunk 1 above)
Comment (drvinceknight) [inline axelrod/match_generator.py]: Not sure what that means but I'm also puzzled by my change... I have no idea how I did that and will blame my editor... :)

Comment (marcharper) [inline axelrod/match_generator.py]: It means "why not this one (as well)"

**Pull request merged**